packages/world/src/systems/admin/AdminSystem.sol

```diff
@@ -37,27 +34,17 @@ contract AdminSystem is System {
 
   function adminAddToInventory(EntityId owner, ObjectTypeId objectTypeId, uint16 numObjectsToAdd) public onlyAdmin {
     require(!objectTypeId.isTool(), "To add a tool, you must use adminAddToolToInventory");
-    addToInventory(owner, ObjectType._get(owner), objectTypeId, numObjectsToAdd);
+    InventoryUtils.addObject(owner, objectTypeId, numObjectsToAdd);
   }
 
   function adminAddToolToInventory(EntityId owner, ObjectTypeId toolObjectTypeId) public onlyAdmin returns (EntityId) {
-    return addToolToInventory(owner, toolObjectTypeId);
+    EntityId tool = createEntity(toolObjectTypeId);
+    InventoryUtils.addEntity(owner, tool);
+    return tool;
   }
 
-  function adminRemoveFromInventory(EntityId owner, ObjectTypeId objectTypeId, uint16 numObjectsToRemove)
```

Comment:
Why were the remove utils removed?

Reply:
ah yeah meant to add those back later, can do it now with the new utils